### PR TITLE
Reduce file size

### PR DESCRIPTION
**Pull request by jbydeley-d2l**
Vue has 2 different builds it comes with. Runtime + Compiler and Runtime Only. Since we have no need for the compiler with how we use Vue, switching to the Runtime Only saves us about 30kb. 
https://vuejs.org/v2/guide/installation.html#Runtime-Compiler-vs-Runtime-only

I've also updated us to the latest version of Vue (2.5.4) which comes with many bug fixes and new features (which slightly increases the file size). After all things, the file size went from 171kb to 154kb.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,10 @@
     "test": "karma start"
   },
   "dependencies": {
-    "@kazupon/vue-i18n-loader": "^0.2.1",
     "axios": "^0.16.2",
-    "vue": "^2.3.3",
-    "vue-i18n": "^7.2.0",
-    "vuex": "^2.4.0"
+    "vue": "^2.5.4",
```
Comment (scasagrande-d2l) [inline package.json]: any behaviour problems with the update to 2.5 ?

Comment (jbydeley-d2l) [reply package.json]: I found none. Tried on Chrome and IE doing the usual things. Would love to have someone sanity check it before I push it though.

**Comment by jbydeley-d2l** [comment]
So after doing some experimenting with webpack and babel, I was able to add async / await polyfills for a few kb. imo, it's worth it since the code it cleans up becomes much much easier to read and understand. If I get any disagreement, I can undo that commit and leave it as it was.

**Comment by RyanSweetD2L** [comment]
:shipit:

**Pull request merged**